docs/operating-scylla/procedures/cluster-management/add-node-to-cluster.rst

```diff
@@ -5,49 +5,73 @@ Adding a New Node Into an Existing ScyllaDB Cluster (Out Scale)
 When you add a new node, other nodes in the cluster stream data to the new node. This operation is called bootstrapping and may
 be time-consuming, depending on the data size and network bandwidth. If using a :ref:`multi-availability-zone <faq-best-scenario-node-multi-availability-zone>`, make sure they are balanced.
```

Comment:
The following 
"If using multi-availability-zone, make sure they are balanced" has no context.
Why should it be balnced? how to make it balance?